esphome/components/template/alarm_control_panel/template_alarm_control_panel.h

```diff
@@ -21,14 +21,30 @@ enum BinarySensorFlags : uint16_t {
   BINARY_SENSOR_MODE_NORMAL = 1 << 0,
   BINARY_SENSOR_MODE_BYPASS_ARMED_HOME = 1 << 1,
   BINARY_SENSOR_MODE_BYPASS_ARMED_NIGHT = 1 << 2,
+  BINARY_SENSOR_MODE_CHIME = 1 << 3,
 };
+
+enum AlarmSensorType : uint16_t {
+  ALARM_SENSOR_TYPE_DELAYED = 0,
+  ALARM_SENSOR_TYPE_INSTANT,
+  ALARM_SENSOR_TYPE_INTERIOR_FOLLOWER
+};
+
 #endif
 
 enum TemplateAlarmControlPanelRestoreMode {
   ALARM_CONTROL_PANEL_ALWAYS_DISARMED,
   ALARM_CONTROL_PANEL_RESTORE_DEFAULT_DISARMED,
 };
 
+using SensorDataStore = struct SensorDataStore { bool last_chime_state; };
+
+using SensorInfo = struct SensorInfo {
+  uint16_t flags;
+  uint16_t type;
```

Comment:
Please use the enum here correctly
```suggestion
  AlarmSensorType type;
```

Reply:
All suggested changes made.